pkg/detectors/easyinsight/easyinsight.go

```diff
@@ -38,20 +40,22 @@ func (s Scanner) Keywords() []string {
 func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
 	dataStr := string(data)
 
-	matches := keyPat.FindAllStringSubmatch(dataStr, -1)
+	keyMatches := keyPat.FindAllStringSubmatch(dataStr, -1)
 
-	idmatches := idPat.FindAllStringSubmatch(dataStr, -1)
+	idMatches := idPat.FindAllStringSubmatch(dataStr, -1)
 
-	for _, match := range matches {
-		if len(match) != 2 {
-			continue
-		}
-		resMatch := strings.TrimSpace(match[1])
-		for _, idmatch := range idmatches {
-			if len(idmatch) != 2 {
+	for _, keyMatch := range keyMatches {
+		resMatch := strings.TrimSpace(keyMatch[1])
+
+		for _, idMatch := range idMatches {
+			resIdMatch := strings.TrimSpace(idMatch[1])
+			/*
```

Comment:
**nit**: Let's keep the commenting style consistent -- `//` instead of `/*` blocks